### DISCUSSION TITLE
HistoryPolicy class name in wiki has wrong package?

### DISCUSSION THREAD

**Discussion by scottkurz** [eclipse-ee4j/eclipselink#2007] [General]
I just got a question about the example in  https://wiki.eclipse.org/EclipseLink/Examples/JPA/History using:

> import org.eclipse.persistence.descriptors.history.HistoryPolicy;

which I believe should just be 

> org.eclipse.persistence.history.HistoryPolicy;
 

Thought about updating the Wiki but wondering..... what if this is the old package name?  So thought I should ask here first...

**Comment by rfelcman** [comment]
Hello,
   yes there is wrong package name. Right one is `org.eclipse.persistence.history.HistoryPolicy` as e.g. https://github.com/eclipse-ee4j/eclipselink/blob/master/jpa/eclipselink.jpa.testapps/jpa.test.advanced/src/main/java/org/eclipse/persistence/testing/models/jpa/advanced/AdvancedHistoryCustomizer.java

**Reply by scottkurz** [reply]
Thx.  I made the edit, pending review.
